sys/include/net/ng_ipv6.h

```diff
@@ -22,14 +28,66 @@
 #ifndef NG_IPV6_H_
 #define NG_IPV6_H_
 
+#include "kernel_types.h"
+#include "net/ng_netbase.h"
+#include "thread.h"
+
 #include "net/ng_ipv6/addr.h"
 #include "net/ng_ipv6/hdr.h"
+#include "net/ng_ipv6/nc.h"
 #include "net/ng_ipv6/netif.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+/**
+ * @brief   Default stack size to use for the IPv6 thread
+ */
+#ifndef NG_IPV6_STACK_SIZE
+#define NG_IPV6_STACK_SIZE (KERNEL_CONF_STACKSIZE_DEFAULT)
+#endif
+
+/**
+ * @brief   Default name for the IPv6 thread
+ */
+#ifndef NG_IPV6_PRIO
+#define NG_IPV6_PRIO            (PRIORITY_MAIN - 3)
+#endif
+
+/**
+ * @brief   Default message queue size to use for the IPv6 thread.
+ */
+#ifndef NG_IPV6_MSG_QUEUE_SIZE
+#define NG_IPV6_MSG_QUEUE_SIZE  (8U)
```

Comment:
It's basically my go-to number for message queue size. But https://lists.riot-os.org/pipermail/devel/2015-April/002458.html seems to indicate it is a good one in this context.